modules/ROOT/pages/6.0-release-notes-core-changes.adoc

```diff
@@ -0,0 +1,297 @@
+
+| `tocupdate`                         | `tableofcontentsupdate`          | Toolbar item                        | This presents in the toolbar’s tooltip text. NB: _Table of Contents_ is now a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].                                                                                                                                                              | see xref:new-and-improved-plugins-table-of-contents[Table of Contents]
+
+| `toc_class`                         | `tableofcontents_class`          | Option                              | NB: _Table of Contents_ is now a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].                                                                                                                                                                                                           | see xref:new-and-improved-plugins-table-of-contents[Table of Contents]
+
+| `toc_depth`                         | `tableofcontents_depth`          | Option                              | NB: _Table of Contents_ is now a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].                                                                                                                                                                                                           | see xref:new-and-improved-plugins-table-of-contents[Table of Contents]
+
+| `toc_header`                        | `tableofcontents_header`         | Option                              | NB: _Table of Contents_ is now a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].                                                                                                                                                                                                           | see xref:new-and-improved-plugins-table-of-contents[Table of Contents]
+|===
+
+- *Commands* are what is passed via the `theeditor.execCommand()` API.
++
+Where a Command name has change, calls to `theeditor.execCommand()` API must be changed to match the new name.
++
+- Configuration *Options* are what is passed when initialising the {productname} editor via `tinymce.init`.
++
+Where an Option name has changed, configurations using that option must be changed to match the new name.
++
+- *Menu items* and *Toolbar items* are *Options* from `tinymce.init` for UI and UX features, such as the {productname} Menu, Toolbar and Contextual Menu.
```

Comment:
Should these link to the relevant docs pages for these options?

Reply:
Haven't looked at this on the branch build to get context but: I've seen a few places we should possibly do that. I'm thinking maybe come back to it later? Seems like if we add some, we might find others, and it'll snowball.

Reply:
Deferring